app/views/shared/_navigation.html.erb

```diff
@@ -14,9 +14,13 @@
     <img src="https://voices-of-consent-static-images.s3.amazonaws.com/Design+Stage.svg" title="Design">
     <span>Design</span>
   </a>
+  <a class="nav-icon" href="http://localhost:3000/box_requests?utf8=%E2%9C%93&filter_by=designed">
```

Comment:
The hostname should not be hard coded because this link will break if we deploy it to remote heroku servers. Since this is an ERB file, you can use the Rails view helpers to generate links that will get dynamically evaluated. There are some examples here: https://github.com/rubyforgood/voices-of-consent/blob/208c5d0ea9b2e89912d2c05e530d86eb128236cb/app/views/boxes/index.html.erb#L31-L33